gc.go

```diff
@@ -33,7 +33,87 @@ type Config struct {
 	WatchDir              string `json:"watch_dir"`
 }
 
+func setupConfig() *Config {
+	var config Config
+
+	configFiles := []string{"${XDG_CONFIG_HOME}/gc/config",
+		"${HOME)/.config/gcrc",
+		"${HOME}/.gcrc"}
+
+	var path string
+	for _, path = range configFiles {
```

Comment:
You don't need to declare `path` above, just `for _, path := range configFiles {`